osu.Server.Queues.ScorePump/Performance/PerformanceMain.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using McMaster.Extensions.CommandLineUtils;
+using osu.Server.Queues.ScorePump.Performance.Totals;
+using osu.Server.Queues.ScorePump.Performance.Values;
+
+namespace osu.Server.Queues.ScorePump.Performance
+{
+    [Command(Name = "update-pp", Description = "Computes the performance (pp) of scores.")]
+    [Subcommand(typeof(UpdateValuesMain))]
+    [Subcommand(typeof(UpdateTotalsMain))]
+    public sealed class PerformanceMain
```

Comment:
`UpdatePerformanceCommand`?

Reply:
I named it this way since there's `PerformanceCommand` too, which is the basis for all other commands. This is only an entrypoint... Would you instead prefer I nest (private) these classes but keep the naming?

Reply:
`PerformanceCommands`? That's how I named such cases these in ASS (`PrilivegedCommands` etc.).

Reply:
Sure, sounds good.